patches/@cosmjs+amino+0.28.11.patch

```diff
@@ -0,0 +1,90 @@
+diff --git a/node_modules/@cosmjs/amino/build/signdoc.js b/node_modules/@cosmjs/amino/build/signdoc.js
```

Comment:
Patch file was created by copying over the following files from https://github.com/regen-network/cosmjs/tree/0.28 into `node_modules/@cosmjs/amino/build`:
- packages/amino/build/signdoc.js
- packages/amino/build/signdoc.js.map
- packages/amino/build/signdoc.d.ts
- packages/amino/build/signdoc.spec.js
- packages/amino/build/signdoc.spec.js.map
- packages/amino/build/signdoc.spec.d.ts

and then running `npx patch-package @cosmjs/amino` from root of regen-js repo.